samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/logic/base/AbstractComponentFacade.java

```diff
@@ -55,8 +54,8 @@ public AbstractComponentFacade() {
 
   /**
    * Creates a {@link Map} with all {@link GenericEntity entities} from the given {@link Collection} using their
-   * {@link GenericEntity#getId() ID} as key. All {@link GenericEntity entities} without an
-   * {@link GenericEntity#getId() ID} ({@code null}) will be ignored.
+   * {@link GenericEntity#getId() ID} as key. All {@link GenericEntity entities} without an {@link GenericEntity#getId()
```

Comment:
Generell gibt es in deinem PR relativ viele "Code-style"-Änderungen. Ich möchte vermeiden, dass wir häufige Changes wegen unterschiedlicher Code-Style-Configs haben. Ist in deinem Eclipse alles "richtig" konfiguriert, nutzt du ggf. eine andere Version, als bisher?